packages/yourloops/test/integration/hcp-home-page.spec.tsx

```diff
@@ -0,0 +1,129 @@
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { Router } from 'react-router-dom'
+import { AuthContextProvider } from '../../lib/auth'
+import { MainLobby } from '../../app/main-lobby'
+import React from 'react'
+import { createMemoryHistory } from 'history'
+import { mockAuth0Hook } from './utils/mockAuth0Hook'
+import { mockNotificationAPI } from './utils/mockNotificationAPI'
+import { mockTeamAPI } from './utils/mockTeamAPI'
+import { mockUserDataFetch } from './utils/auth'
+import { mockPatientAPI, monitoredPatient, removePatientMock, unMonitoredPatient } from './utils/mockPatientAPI'
+import { act, render, screen, within } from '@testing-library/react'
+import { checkHeader } from './utils/header'
+import { checkDrawer } from './utils/drawer'
+import { checkFooter } from './utils/footer'
+import { mockDataAPI } from './utils/mockDataAPI'
+import { mockDirectShareApi } from './utils/mockDirectShareAPI'
+import PatientAPI from '../../lib/patient/patient-api'
+
+describe('HCP remove a patient from list', () => {
+  const firstName = 'Eric'
+  const lastName = 'Ard'
+  beforeAll(() => {
+    mockAuth0Hook()
+    mockNotificationAPI()
+    mockTeamAPI()
+    mockUserDataFetch(firstName, lastName)
+    mockPatientAPI()
+    mockDataAPI()
+    mockDirectShareApi()
+  })
+
+  function getHomePage() {
+    const history = createMemoryHistory({ initialEntries: ['/'] })
+    return (
+      <Router history={history}>
+        <AuthContextProvider>
+          <MainLobby />
+        </AuthContextProvider>
+      </Router>
+    )
+  }
+
+  it('should render the home page with correct components', async () => {
+    await act(async () => {
+      render(getHomePage())
+    })
+    checkHeader(`${firstName} ${lastName}`)
+    checkDrawer()
+    checkFooter()
+    expect(screen.getByTestId('patients-secondary-bar')).toBeVisible()
+    expect(screen.getByRole('table')).toBeVisible()
+  })
+
+  it('should display a list of patient and be able to remove one of them', async () => {
+    await act(async () => {
+      render(getHomePage())
+    })
+
+    expect(screen.queryAllByLabelText('flag-icon-active')).toHaveLength(0)
+    expect(screen.getAllByLabelText('flag-icon-inactive')).toHaveLength(2)
+
+    const patientRow = screen.queryByTestId(`patient-row-${unMonitoredPatient.userId}`)
+    const removeButton = within(patientRow).getByRole('button', { name: 'Remove patient-ylp.ui.test.patient28@diabeloop.fr' })
+    expect(removeButton).toBeInTheDocument()
+
+    removeButton.click()
+    const removeDialog = screen.getByRole('dialog')
+    expect(removeDialog).toBeInTheDocument()
+    const confirmRemoveButton = within(removeDialog).getByRole('button', { name: 'Remove patient' })
```

Comment:
More content of the remove dialog can be tested (at least the title/subtitle and which team is selected in the case of the patient having a single team)